plugin/v2/settable.go

```diff
@@ -0,0 +1,89 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
+	value string
+}
+
+var (
+	allowedSettableFieldsEnv     = []string{"value"}
+	allowedSettableFieldsArgs    = []string{"value"}
+	allowedSettableFieldsDevices = []string{"path"}
+	allowedSettableFieldsMounts  = []string{"source"}
+)
+
+func newSettables(args []string) []settable {
+	settables := make([]settable, 0, len(args))
+	for _, arg := range args {
+		settables = append(settables, newSettable(arg))
+	}
+	return settables
+}
+
+func newSettable(arg string) settable {
```

Comment:
`name=` seems a valid case to me. by extension even `name` could be valid, to set the env to `""`